operators/src/processing/meteosat/mod.rs

```diff
@@ -5,28 +5,28 @@ mod reflectance;
 mod satellite;
 mod temperature;
 
-fn slope_key() -> RasterPropertiesKey {
+fn new_slope_key() -> RasterPropertiesKey {
```

Comment:
can those be `const` functions, or just constants? They are always the same

Reply:
i guess we will have them as options the user can set in a next version. So i think we should leave it like this. They can't be const since Strings are on the heap